Gemfile

```diff
@@ -28,6 +28,7 @@ gem "activerecord-virtual_attributes", "~>3.0.0"
 gem "activerecord-session_store",     "~>1.1"
 gem "acts_as_tree",                   "~>2.7" # acts_as_tree needs to be required so that it loads before ancestry
 gem "ancestry",                       "~>3.0.7",       :require => false
+gem "autosde_openapi_client",         "=1.0.3"
```

Comment:
You can actually put this in the autosde plugin's gemspec: https://github.com/ManageIQ/manageiq-providers-autosde/blob/master/manageiq-providers-autosde.gemspec#L22

> "=1.0.3"

Best to use something like `~>1.0` so that the gem can be updated without having to bump the `.z` component.

Reply:
@borisko123 I agree with @agrare, please close this PR, and make all the required modifications in autosde provider (as part of https://github.com/ManageIQ/manageiq-providers-autosde/pull/52)